src/test/developTest/java/mServer/crawler/sender/wdr/WdrVideoDetailsDeserializerTest.java

```diff
@@ -0,0 +1,85 @@
+            { "/wdr/wdr_video_details2.html", "Ausgerechnet", "Ausgerechnet - Schokolade", "Knapp 25 Prozent der Bevölkerung verzehren Schokolade mehrmals in der Woche. Hinzu kommt gut ein weiteres Viertel, das Schokolade etwa einmal pro Woche genießt. Frauen greifen hier generell häufiger zu als Männer.", "http://www1.wdr.de/mediathek/video/sendungen/ausgerechnet/video-ausgerechnet---schokolade-102.html", "http://wdradaptiv-vh.akamaihd.net/i/medp/ondemand/weltweit/fsk0/140/1407842/,1407842_16309723,1407842_16309728,1407842_16309725,1407842_16309726,1407842_16309724,1407842_16309727,.mp4.csmil/index_2_av.m3u8", "196|0_av.m3u8", "196|4_av.m3u8", "http://wdrmedien-a.akamaihd.net/medp/ondemand/weltweit/fsk0/140/1407842/1407842_16348809.xml", "15.07.2017", "16:00:00", "00:43:35", "http://deviceids-medp.wdr.de/ondemand/140/1407842.js", "/wdr/wdr_video2.js", "http://wdradaptiv-vh.akamaihd.net/i/medp/ondemand/weltweit/fsk0/140/1407842/,1407842_16309723,1407842_16309728,1407842_16309725,1407842_16309726,1407842_16309724,1407842_16309727,.mp4.csmil/master.m3u8", "/wdr/wdr_video2.m3u8" },
+        });
+    }
+
+    private final String htmlFile;
+    private final String expectedTheme;
+    private final String expectedTitle;
+    private final String expectedDescription;
+    private final String expectedWebsite;
+    private final String expectedVideoUrlSmall;
+    private final String expectedVideoUrlNormal;
+    private final String expectedVideoUrlHd;
+    private final String expectedSubtitle;
+    private final String expectedDate;
+    private final String expectedTime;
+    private final String expectedDuration;
+    
+    private final WdrVideoDetailsDeserializer target;
+    
+    public WdrVideoDetailsDeserializerTest(String aHtmlFile, String aTheme, String aTitle, String aDescription, String aWebsite, String aVideoUrlNormal, String aVideoUrlSmall, String aVideoUrlHd, String aSubtitle, String aDate, String aTime, String aDuration, String aJsUrl, String aJsFile, String aM3u8Url, String aM3u8File) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 16 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)